Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.h

```diff
@@ -89,23 +109,13 @@ class MediaPlaybackTargetContext {
         return m_outputContext;
     }
 
-    RetainPtr<NSData> serializedOutputContext() const
-    {
-        ASSERT(m_type == Type::SerializedAVOutputContext);
-        return m_serializedOutputContext;
-    }
-
-    WEBCORE_EXPORT bool serializeOutputContext();
-    WEBCORE_EXPORT bool deserializeOutputContext();
-
-    bool encodingRequiresPlatformData() const { return m_type == Type::AVOutputContext || m_type == Type::SerializedAVOutputContext; }
+    bool encodingRequiresPlatformData() const { return m_type == Type::AVOutputContext; }
```

Comment:
Seems confusing to have both this method and `m_requirePlatformData` with such similar names but being different things.